### ISSUE TITLE
Screen Reader Support For Blind Spymaster

### ISSUE THREAD

**Issue by ertay** [ninjabunny/KodeNames#18]
I was just testing the app with a screen reader on my phone (iPhone) and I noticed that there is no information whether a word is blue, red, bystander or assassin when going through the words. So, a blind player cannot play as the spymaster. Any chance you can add this feature? I'm not sure if you can simply add alt-text to the words that hte screen reader picks up while it is not visible in the UI to distract sighted players.

**Comment by ninjabunny** [comment]
This should not be so difficult to implement. Let me look into it.

**Comment by ertay** [comment]
@ninjabunny 

Was this issue solved? I just checked the app and you still can't tell with the screen reader whether a word is red, blue, bystander, or assassin.

**Comment by ninjabunny** [comment]
Hey, sorry for taking so long. My life was pretty hectic for the last year. now covid allowed me more time to fix this issue. I just forgot to push it. please test it out now and see if it reads for you. i forgot how to use mac screen reader on the site.

**Comment by ertay** [comment]
Hi,

No worries, hope you are doing well during these chaotic times. Unfortunately, it still doesn't seem to work. Both in the Spymaster view and also in the normal view when you click the links to reveal the colors, nothing besides the word is read. Tried with both NVDA and Narrator on Windows. Maybe you should look into ARIA labels? These are visually invisible but the screen reader reads the values. You'll just need to enable them and disable them depending on the state of the word.

Thanks for doing this, I have a blind board gamers group which might be interested in playing Codenames as we are all stuck at home right now :)
